waku/v2/node/waku_node.nim

```diff
@@ -427,10 +427,12 @@ proc filterSubscribe*(node: WakuNode, pubsubTopic: PubsubTopic, contentTopics: C
     error "cannot register filter subscription to topic", error="waku filter client is nil"
     return
 
-  let remotePeer = when peer is string: parseRemotePeerInfo(peer)
-                   else: peer
+  var remotePeer = parsePeerInfo(peer)
```

Comment:
Any reason this is a `var` and not `let`?

Reply:
ah yes, you are right, thanks !